client/src/modules/Registry/components/Form/DoneSection/index.tsx

```diff
@@ -0,0 +1,60 @@
+import { Card, Image, Row, Skeleton, Typography } from 'antd';
+import { SUCCESS_TEXT } from 'modules/Registry/constants';
+
+const { Title, Paragraph } = Typography;
+const arrowSvg = (
```

Comment:
Have you checked proper or similar Icon from https://ant.design/components/icon?

Reply:
is there a way to make that big blue arrow from small gray by setting props on antd icon? Seams legit to me adding svg arrow from figma
![image](https://user-images.githubusercontent.com/67139199/215713461-984ba744-f6ec-4e0a-86dc-20301f9e916c.png)
![image](https://user-images.githubusercontent.com/67139199/215714591-1440414e-52d6-476c-b4dc-9eb325c36e24.png)

Reply:
It there is no proper built-in icon, you can add Custom Icon https://ant.design/components/icon#components-icon-demo-custom - find examples in repository

Reply:
fixed